### PR TITLE
system-variables: update from generated source

### PR DESCRIPTION
**Pull request by morgo**
### What is changed, added or deleted? (Required)

This updates the system-variables documentation, automatically generated from the source code.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

rand_seed from https://github.com/pingcap/tidb/pull/29936
backoff_lock_fast is from https://github.com/tikv/client-go/pull/388
tidb_store_limit from https://github.com/pingcap/tidb/pull/30522

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by dveeden — APPROVED** [review]

**Review by Defined2014 — APPROVED** [review]

File: system-variables.md [hunk 1]
```diff
@@ -463,7 +479,7 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 ### tidb_backoff_lock_fast
 
 - Scope: SESSION | GLOBAL
-- Default value: `100`
+- Default value: `10`
```
Comment (TomShawn) [inline system-variables.md:482]: @seiya-annie PTAL this default value change. From which version this value has been changed?

Comment (seiya-annie) [reply system-variables.md:482]: it's still 100 in v5.3.0, maybe a new change for master

Comment (morgo) [reply system-variables.md:482]: In the checklist at the time there was no option, but I selected master.

This affects TiDB 5.4 and above.

**Review by seiya-annie — APPROVED** [review]

**Pull request merged**